packages/cadl-ranch-specs/http/azure/resource-manager/resources/extension.tsp

```diff
@@ -0,0 +1,544 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Resources;
+
+model ExtensionsResource is ExtensionResource<ExtensionsResourceProperties> {
+  ...ResourceNameParameter<ExtensionsResource>;
+}
+
+/** ExtensionsResource properties */
+model ExtensionsResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+
+  /** The status of the last operation. */
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+/** The interface of extensions resources,
+ * it contains 4 kinds of URLs (resource, resource group, subscription and tenant)
+ */
+@armResourceOperations
+interface ExtensionsResources {
+  @scenario
+  @scenarioDoc("""
+      Resource GET operation.
```

Comment:
```suggestion
      Resource GET operation for tenant level extension resource.
```

Reply:
same for other `Resource XX operation`. please add extension kind

Reply:
Fixed in the new version.